kubernixos.go

```diff
@@ -232,3 +236,19 @@ func askForConfirmation() bool {
 		return askForConfirmation()
 	}
 }
+
+func eval(attribute string) (*bytes.Buffer, error) {
+	if flake {
+		return nix.FlakeEval(attribute)
+	} else {
+		return nix.Eval(attribute, nixArgs)
+	}
+}
+
+func build(attribute string) (string, error) {
+	if flake {
+		return nix.FlakeBuild(attribute)
+	} else {
+		return nix.Build(attribute, nixArgs)
+	}
+}
```

Comment:
Not really sure how to do it better, but I think it's not-so-nice that different functions rely on the global vars defined in the module. otoh it requires a lot of changes to pass them down where they're needed, so.. Yeah.

Reply:
Yeah, I wasn't very happy about it either. I saw that this was already vaguely grabbed from global values in the existing invocations, so I figure I've only made things slightly "better" in the flakeful case and exactly the same in the "old style" case.